detectorSegmentations/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+#################################################################################
+##Package : detectorSegmentations
+#################################################################################
+
+file(GLOB sources src/*.cpp)
+
+add_library(detectorSegmentations SHARED ${sources} )
+dd4hep_generate_rootmap(detectorSegmentationsPlugin)
+target_link_libraries(detectorSegmentations DD4hep::DDCore)
+target_include_directories(detectorSegmentations
+    PUBLIC
+        $<INSTALL_INTERFACE:include>
+        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+)
+
+file(GLOB headers include/detectorSegmentations/*.h)
+set_target_properties(detectorSegmentations PROPERTIES PUBLIC_HEADER "${headers}")
+set_property(TARGET detectorSegmentations PROPERTY GLOBAL)
```

Comment:
Does this do anything? I don't see that GLOBAL is a property for targets:
https://cmake.org/cmake/help/latest/manual/cmake-properties.7.html#properties-on-targets